tmk_core/protocol/chibios/usb_main.c

```diff
@@ -103,7 +103,7 @@ static const USBDescriptor *usb_get_descriptor_cb(USBDriver *usbp, uint8_t dtype
     static USBDescriptor desc;
     uint16_t             wValue = ((uint16_t)dtype << 8) | dindex;
     desc.ud_string              = NULL;
-    desc.ud_size                = get_usb_descriptor(wValue, wIndex, (const void **const) & desc.ud_string);
+    desc.ud_size                = get_usb_descriptor(wValue, wIndex, (const void **const)&desc.ud_string);
```

Comment:
Running clang-format within the docker container puts this back to how it wasw
```suggestion
    desc.ud_size                = get_usb_descriptor(wValue, wIndex, (const void **const) & desc.ud_string);
```

Reply:
Interesting could you tell me the clang-format version? Mine is 12.0.0.

Reply:
Whatever is in <https://hub.docker.com/r/qmkfm/base_container>, which is the "official" environment.

Reply:
which is an early version of 11, IIRC.

Reply:
The dockerfile in qmk_base_container says  `clang-format-7` rather old. Anyways I'll get that version and apply the formatting of that.

Reply:
Applied clang-format-7 locally.